packages/bot-cleaning/src/MarketCleaner.ts

```diff
@@ -146,39 +146,22 @@ export class MarketCleaner {
         this.#whitelistedDustCleaningMaker &&
         this.#whitelistedDustCleaningMaker.has(offer.maker.toLowerCase())
       ) {
-        const inboundTkn =
-          ba == "bids" ? semibook.market.base : semibook.market.quote;
         const outboundTkn =
           ba == "bids" ? semibook.market.quote : semibook.market.base;
 
         const takerWants = new Big(1).div(
           new Big(10).pow(outboundTkn.decimals)
         );
-        const takerGives = new Big(1).div(new Big(10).pow(inboundTkn.decimals));
 
         cleaningPromises.push(
-          this.#cleanOffer(
-            offer,
-            ba,
-            gasPrice,
-            takerWants,
-            takerGives,
-            contextInfo
-          ) // takerWants: outboundTkn, takerGives: inboundTkn
+          this.#cleanOffer(offer, ba, gasPrice, takerWants, contextInfo) // takerWants: outboundTkn, takerGives: inboundTkn
```

Comment:
Would be nice if you could change the logic so that we first do a big multicall in which we check if offers are failling if they are then we do an estimateGas plus call to clean the offer